staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -59,6 +59,19 @@ type Versioner interface {
 	ParseListResourceVersion(resourceVersion string) (uint64, error)
 }
 
+// Obfuscator represents a symmetric encryption algorithm. he value of the output of this
```

Comment:
Does this interface need to live in the storage package? Maybe it can stay in the etcd package. We can always promote it if we need to add a storage backend with the same need.

Reply:
That is what I was thinking too, but there wasn't an interfaces.go in the etcd package and I wasn't sure if I could just create one. Definitely doesn't need to be in the storage package.